### PR TITLE
mostly formatting

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/functions-ii/function-constructor-vs-function-expression.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/functions-ii/function-constructor-vs-function-expression.md?ref=a73d46bd988377de257155071d170d42036e6faf)

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 1]
```diff
@@ -1,41 +1,16 @@
...
   - '[developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#Function_constructor_vs._function_declaration_vs._function_expression){website}'
 
-
 aspects:
   - introduction
 
-
 ---
 
 # Function constructor vs. function expression
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]:
```suggestion
# Function Constructor vs. Function Expression
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 2]
```diff
@@ -1,41 +1,16 @@
...
-
-  javascript.functions.4: 10
-
-tags:
-
-  - introduction
-
 links:
-
   - '[developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#Function_constructor_vs._function_declaration_vs._function_expression){website}'
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]:
```suggestion
  - '[Constructor vs. Declaration vs. Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#Constructor_vs._declaration_vs._expression){documentation}'
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 3]
```diff
@@ -47,27 +22,27 @@ The concepts of constructors, declaration and expressions are often misused. Her
 
 A function defined with the `function` *constructor* assigned to the variable `multiply`:
 
-```
+```js
 function multiply(x, y) {
-   return x * y;
+  return x * y;
 }
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]: removed extra newline
```suggestion
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 4]
```diff
@@ -47,27 +22,27 @@ The concepts of constructors, declaration and expressions are often misused. Her
...
 ```
 
 Function *expression* of an anonymous function assigned to the variable `multiply`:
 
 ```
-var multiply = function(x, y) {
-   return x * y;
+let multiply = function(x, y) {
+  return x * y;
 }
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]: removed extra newline
```suggestion
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 5]
```diff
@@ -47,27 +22,27 @@ The concepts of constructors, declaration and expressions are often misused. Her
...
 ```
 
 Function *expression* of a `function` named `mult` assigned to the variable `multiply`:
 
 ```
-var multiply = function mult(x, y) {
-   return x * y;
+let multiply = function mult(x, y) {
+  return x * y;
 };
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]: removed extra newline
```suggestion
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 6]
```diff
@@ -78,8 +53,8 @@ var multiply = function mult(x, y) {
 What is the following code? ???
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]: should we add the `???` after the codeblock?
```suggestion
What is the following code?
```

File: javascript/core/functions-ii/function-constructor-vs-function-expression.md [hunk 7]
```diff
@@ -78,8 +53,8 @@ var multiply = function mult(x, y) {
...
 
 ```javascript
-var multiply = function times(x, y) {
-   return x * y;
+let multiply = function times(x, y) {
+  return x * y;
 };
 ```
```
Comment (kapnobatai137) [inline javascript/core/functions-ii/function-constructor-vs-function-expression.md]:
``````suggestion
```

???
``````

**Comment by kapnobatai137** [comment]
image
we could probably add some more to this insight

**Comment by nem035** [comment]
Good to merge? @kapnobatai137 @Stefan-Stojanovic

**Pull request merged**